package.json

```diff
@@ -36,8 +40,11 @@
     "moment-timezone": "^0.5.33",
     "query-string": "^7.0.1",
     "react": "^16.13.1",
+    "react-autosuggest": "^10.1.0",
     "react-dom": "^16.13.1",
     "react-helmet-async": "^1.1.2",
+    "react-instantsearch-dom": "^6.25.0",
+    "react-instantsearch-hooks-web": "^6.25.0",
     "react-moment": "^0.9.6",
     "refractor": "^2.10.0",
     "rxjs": "^6.5.2",
```

Comment:
Could you delete the `package-lock.json` file from the repo? We use `yarn.lock` to lock dependency versions, but package-lock is for npm instead of yarn. :+1: